Source/multi.cpp

```diff
@@ -333,14 +332,13 @@ void ProcessTmsgs()
 void SendPlayerInfo(int pnum, _cmd_id cmd)
 {
 	static_assert(alignof(PlayerPack) == 1, "Fix pkplr alignment");

```

Comment:
```suggestion
	static_assert(alignof(PlayerPack) == 1, "Fix pPack alignment");
```

Also is that still needed?

Reply:
No. Fixed.

Reply:
Isn't it important for the `reinterpret_cast<byte *>(&pPack)`?

Reply:
It would only matter if `alignof(byte)` were larger than `alignof(PlayerPack)`.